TAScheduler/models.py

```diff
@@ -23,16 +23,29 @@ class Section(models.Model):
     credits = models.IntegerField()
 
 
+# noinspection DuplicatedCode
 class TA(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE, null=True)
     sections = models.ManyToManyField(Section)
 
 
+# noinspection DuplicatedCode
+class TAToSection(models.Model):
+    ta = models.ForeignKey(TA, on_delete=models.CASCADE, null=True)
+    section = models.ForeignKey(Section, on_delete=models.CASCADE, null=True)
+
+
+# noinspection DuplicatedCode
 class Instructor(models.Model):
```

Comment:
Instructor needs "maximum assignments". for example, it doesn't make sense for an instructor to be assigned to 100 courses - set it around ~5,6

Reply:
Why does the instructor need a maximum assignments? I feel that is a little redundant

Reply:
It can't be redundant if it isn't included anywhere - my state diagram specifies this so I think it will be a good idea to include it in the tables (we can remove it later if it's not needed or too much hassle)